src/couch/src/couch_att.erl

```diff
@@ -734,7 +734,7 @@ upgrade_encoding(Encoding) -> Encoding.
 
 
 max_attachment_size() ->
-    case config:get("couchdb", "max_attachment_size", "infinity") of
+    case config:get("couchdb", "max_attachment_size", 1024 * 1024 * 1024) of
```

Comment:
Erlang doesn't do compile time optimizations. I think multiplications would be slow. Could you use a define instead.

```
% 1024 * 1024 * 1024
-define(GB, 1073741824).

....
max_attachment_size() ->
    case config:get("couchdb", "max_attachment_size", ?GB) of
```